build.sh

```diff
@@ -4,7 +4,10 @@ set -u # a reference to any variable you have not previously defined causes the
 set -o pipefail # If any command in a pipeline fails, that return code will be used as the return code of the whole pipeline
 
 PROGNAME=$(basename "$0")
-
+ARCH="x86_64"
+if [[ "`uname -m`" =~ "arm64" ]]; then 
+  ARCH="aarch64"
+fi
```

Comment:
I think we could use result from `uname -m` in any case:
```
  ARCH=`uname -m`
```

Reply:
Then it's OK for me.

Reply:
According to https://docs.osgi.org/reference/osnames.html we could change it, but I do not know whether we need it at this point. It's still just a small bash script after all.